the gen_profile sed use is busted so just use gsed in this weird file
--
--- a/seccomp.sh
+++ b/seccomp.sh
@@ -67,13 +67,13 @@
 	__template="$(eval printf '%s' "\${${2}}")"
 	shift; shift
 
-	sed -i "${__line_no}s#.*#${__template}#" "${TMP}"
+	gsed -i "${__line_no}s#.*#${__template}#" "${TMP}"
 
 	IFS=' '
 	for __def in ${@}; do
 		__key="@${__def%%:*}@"
 		__value="${__def#*:}"
-		sed -i "${__line_no}s/${__key}/${__value}/" "${TMP}"
+		gsed -i "${__line_no}s/${__key}/${__value}/" "${TMP}"
 	done
 	unset IFS
 }
@@ -90,7 +90,7 @@
 	for __def in ${@}; do
 		__key="@${__def%%:*}@"
 		__value="${__def#*:}"
-		__out="$(printf '%s' "${__out}" | sed "s#${__key}#${__value}#")"
+		__out="$(printf '%s' "${__out}" | gsed "s#${__key}#${__value}#")"
 	done
 
 	printf '%s\n' "${__out}" >> "${OUT}"
@@ -196,7 +196,7 @@
 		for __node in $(seq 1 ${__level_nodes}); do
 			__cmp_line=$(( __statements_bst + __cmp_pos ))
 			__lr=$(( __ll + 1 ))
-			__nr="$(sed -n ${__cmp_line}p "${TMP}")"
+			__nr="$(gsed -n ${__cmp_line}p "${TMP}")"
 
 			sub ${__line} BST "NR:${__nr}" "L:${__ll}" "R:${__lr}"
 
@@ -216,7 +216,7 @@
 
 	for __leaf in $(seq 1 ${__level_nodes}); do
 		__cmp_line=$(( __statements_bst + __cmp_pos ))
-		__nr="$(sed -n ${__cmp_line}p "${TMP}")"
+		__nr="$(gsed -n ${__cmp_line}p "${TMP}")"
 		sub ${__line} BST "NR:${__nr}" "L:${__ll}" "R:${__lr}"
 
 		__ll=$(( __lr + __distance - 1 ))
@@ -227,7 +227,7 @@
 
 	# Calls
 	for __i in $(seq $(( __statements_bst + 1 )) ${__statements}); do
-		__nr="$(sed -n ${__i}p "${TMP}")"
+		__nr="$(gsed -n ${__i}p "${TMP}")"
 		eval __name="\${__syscall_nr_${__nr}}"
 		__allow=$(( __statements - __i + 1 ))
 		sub ${__i} CALL "NR:${__nr}" "NAME:${__name}" "ALLOW:${__allow}"
@@ -237,9 +237,9 @@
 }
 
 printf '%s\n' "${HEADER}" > "${OUT}"
-__profiles="$(sed -n 's/[\t ]*\*[\t ]*#syscalls:\([^ ]*\).*/\1/p' ${IN} | sort -u)"
+__profiles="$(gsed -n 's/[\t ]*\*[\t ]*#syscalls:\([^ ]*\).*/\1/p' ${IN} | sort -u)"
 for __p in ${__profiles}; do
-	__calls="$(sed -n 's/[\t ]*\*[\t ]*#syscalls\(:'"${__p}"'\|\)[\t ]\{1,\}\(.*\)/\2/p' ${IN})"
+	__calls="$(gsed -n 's/[\t ]*\*[\t ]*#syscalls\(:'"${__p}"'\|\)[\t ]\{1,\}\(.*\)/\2/p' ${IN})"
 	__calls="${__calls} ${EXTRA_SYSCALLS:-}"
 	__calls="$(filter ${__calls})"
 	echo "seccomp profile ${__p} allows: ${__calls}" | tr '\n' ' ' | fmt -t
